src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -19542,42 +19527,15 @@ static record NV
 ";
             var comp = CreateCompilation(text);
             comp.VerifyDiagnostics(
-                // (2,15): error CS0708: 'NV.EqualityContract': cannot declare instance members in a static class
-                // static record NV
-                Diagnostic(ErrorCode.ERR_InstanceMemberInStaticClass, "NV").WithArguments("NV.EqualityContract").WithLocation(2, 15),
-                // (2,15): error CS0722: 'NV': static types cannot be used as return types
-                // static record NV
-                Diagnostic(ErrorCode.ERR_ReturnTypeIsStaticClass, "NV").WithArguments("NV").WithLocation(2, 15),
-                // (2,15): error CS0708: 'GetHashCode': cannot declare instance members in a static class
-                // static record NV
-                Diagnostic(ErrorCode.ERR_InstanceMemberInStaticClass, "NV").WithArguments("GetHashCode").WithLocation(2, 15),
-                // (2,15): error CS0708: 'Equals': cannot declare instance members in a static class
-                // static record NV
-                Diagnostic(ErrorCode.ERR_InstanceMemberInStaticClass, "NV").WithArguments("Equals").WithLocation(2, 15),
-                // (2,15): error CS0708: 'Equals': cannot declare instance members in a static class
-                // static record NV
-                Diagnostic(ErrorCode.ERR_InstanceMemberInStaticClass, "NV").WithArguments("Equals").WithLocation(2, 15),
-                // (2,15): error CS1057: 'NV.NV(NV)': static classes cannot contain protected members
-                // static record NV
-                Diagnostic(ErrorCode.ERR_ProtectedInStatic, "NV").WithArguments("NV.NV(NV)").WithLocation(2, 15),
-                // (2,15): error CS1057: 'NV.EqualityContract': static classes cannot contain protected members
-                // static record NV
-                Diagnostic(ErrorCode.ERR_ProtectedInStatic, "NV").WithArguments("NV.EqualityContract").WithLocation(2, 15),
-                // (2,15): error CS1057: 'NV.EqualityContract.get': static classes cannot contain protected members
+                // (2,15): error CS0106: The modifier 'static' is not valid for this item
```

Comment:
Nice! I didn't expect this would also resolve the cascading diagnostics problem :-)